src/phasorpy/phasor.py

```diff
@@ -444,18 +450,87 @@ def phasor_calibrate(
     (array([-1.56, 1.934, 0.3279]), array([5.985, 10.6, 1.986]))
 
     """
-    phi0 = numpy.asarray(phase0)
-    mod0 = numpy.asarray(modulation0)
-    if phi0.shape != mod0.shape:
-        raise ValueError(f'{phi0.shape=} != {mod0.shape=}')
+    re = numpy.asarray(real)
+    im = numpy.asarray(imag)
+    if re.shape != im.shape:
+        raise ValueError(f'{re.shape=} != {im.shape=}')
```

Comment:
I think the error message should refer to the documented parameter names, `real` and `imag` in this case. The user does not know about `re` and `im`

Reply:
I didn't realize this. Thanks for pointing it out.